src/notification/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { PureComponent, Fragment } from 'react'
+import { canUseDOM } from 'exenv'
+import cc from 'classcat'
+import { CrossIcon } from 'icon'
+import { createPortal } from 'react-dom'
+import { color, space } from '_utils/branding'
+import TransitionGroup from 'react-transition-group/TransitionGroup'
+import CustomTransition, { AnimationType } from 'transitions'
+import Button from 'button'
+
+import style from './style'
+
+export interface NotificationProps {
+  readonly close: () => void
+  readonly isOpen: boolean
+  readonly className?: string
+}
+
+class Notification extends PureComponent<NotificationProps> {
+  private portalNode: HTMLElement
+  static defaultProps: Partial<NotificationProps> = {
+    isOpen: false,
+  }
+
+  constructor(props: NotificationProps) {
+    super(props)
+    if (canUseDOM) {
+      this.portalNode = document.createElement('div')
+      document.body.appendChild(this.portalNode)
+    }
+  }
+
+  componentWillUnmount() {
+    document.body.removeChild(this.portalNode)
+    this.portalNode = null
+  }
+
+  render() {
+    if (!canUseDOM || !this.portalNode) {
+      return null
+    }
+
+    const modalElement = (
+      <TransitionGroup component="div" className="transition-wrapper notification-container">
+        {this.props.isOpen && (
+          <CustomTransition animationName={AnimationType.SLIDE_UP}>
+            <div className={cc(['notification', this.props.className])}>
+              <p className="notification-content">{this.props.children}</p>
+              <Button
+                status={Button.STATUS.UNSTYLED}
+                className={`notification-cross`}
```

Comment:
Is that different from `className="notification-cross"`?

Reply:
nope, good catch